language/move-lang/src/parser/syntax.rs

```diff
@@ -1706,7 +1788,8 @@ fn parse_address_block(tokens: &mut Lexer) -> Result<(Loc, Address, Vec<ModuleDe
     consume_token(tokens, Tok::LBrace)?;
     let mut modules = vec![];
     while tokens.peek() != Tok::RBrace {
-        modules.push(parse_module(tokens)?);
+        let attributes = maybe_parse_attributes(tokens)?;
```

Comment:
This doesn't match your change to the grammar comment. I'm not sure which one you want. The grammar shows an Attributes list before the "address", but the code parses an Attributes list before each module.